server/routes/prisonPersonMigration/prisonPersonMigrationController.ts

```diff
@@ -0,0 +1,64 @@
+import { Request, Response } from 'express'
+import moment from 'moment'
+import NomisMigrationService, { Context } from '../../services/nomisMigrationService'
+import { MigrationHistory } from '../../@types/migration'
+import buildUrl from '../../utils/applicationInsightsUrlBuilder'
+
+interface Filter {
+  prisonerNumber?: string
```

Comment:
Wont you be filtering by prison? If migrations do the whole estate in one go you dont actually need a UI